examples/stable_diffusion_v2/eval/README.md

```diff
@@ -29,8 +29,31 @@ For more usage, please run `python eval/eval_fid.py -h`.
 
 ### CLIP Score
 
-Coming soon
+To compute the CLIP score between images and texts, please run 
 
+- Mindspore backend
+```
+python eval/eval_clip_score.py --load_checkpoint <path-to-model> --image_path <path-to-image> --prompt <string/path-to-txt>
```

Comment:
It would be more clear if we rename `image_path` to `image_path_or_dir` , and `prompt` to `prompt_or_path`.

Reply:
done

Reply:
done